packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -510,7 +511,18 @@ class NetworkImage extends ImageProvider<NetworkImage> {
     );
   }
 
-  static final HttpClient _httpClient = HttpClient();
+  /// Do not access this field directly; use [_httpClient] instead.
+  static final HttpClient _sharedHttpClient = HttpClient()..autoUncompress = false;
```

Comment:
Why disable `autoUncompress` by default with this change?

Reply:
Because I was conflating two related changes.  Fixed 😄